subsys/nrf_security/src/drivers/nrf_oberon/Kconfig

```
@@ -1,353 +1,1114 @@
+										  (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA6144_SHA224_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA6144_SHA256_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA6144_SHA384_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA6144_SHA512_OAEP))) \
+			|| (PSA_WANT_ALG_RSA_PSS && ((PSA_WANT_ALG_SHA_1   && !PSA_ACCEL_RSA6144_SHA1_PSS)   || \
+										 (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA6144_SHA224_PSS) || \
+										 (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA6144_SHA256_PSS) || \
+										 (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA6144_SHA384_PSS) || \
+										 (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA6144_SHA512_PSS)))  \
+			|| (PSA_WANT_ALG_RSA_PKCS1V15_SIGN && ((PSA_WANT_ALG_SHA_1  && !PSA_ACCEL_RSA6144_SHA1_PKCS1V15_SIGN)     || \
+												   (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA6144_SHA224_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA6144_SHA256_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA6144_SHA384_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA6144_SHA512_PKCS1V15_SIGN)))
+
+config PSA_NEED_OBERON_RSA_KEY_SIZE_8192
+	bool
+	default y
+	depends on PSA_WANT_RSA_KEY_SIZE_8192
+	depends on (PSA_WANT_ALG_RSA_PKCS1V15_CRYPT && !PSA_ACCEL_RSA8192_PKCS1V15_CRYPT) \
+			|| (PSA_WANT_ALG_RSA_OAEP && ((PSA_WANT_ALG_SHA_1   && !PSA_ACCEL_RSA8192_SHA1_OAEP)     || \
+										  (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA8192_SHA224_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA8192_SHA256_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA8192_SHA384_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA8192_SHA512_OAEP))) \
+			|| (PSA_WANT_ALG_RSA_PSS && ((PSA_WANT_ALG_SHA_1   && !PSA_ACCEL_RSA8192_SHA1_PSS)   || \
+										 (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA8192_SHA224_PSS) || \
+										 (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA8192_SHA256_PSS) || \
+										 (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA8192_SHA384_PSS) || \
+										 (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA8192_SHA512_PSS)))  \
+			|| (PSA_WANT_ALG_RSA_PKCS1V15_SIGN && ((PSA_WANT_ALG_SHA_1  && !PSA_ACCEL_RSA8192_SHA1_PKCS1V15_SIGN)     || \
+												   (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA8192_SHA224_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA8192_SHA256_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA8192_SHA384_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA8192_SHA512_PKCS1V15_SIGN)))
+
+config PSA_NEED_OBERON_RSA_PSS
+	bool
+	default y
+	depends on PSA_WANT_ALG_RSA_PSS
+	depends on PSA_NEED_OBERON_RSA_KEY_SIZE_1024 || \
+			   PSA_NEED_OBERON_RSA_KEY_SIZE_1536 || \
+			   PSA_NEED_OBERON_RSA_KEY_SIZE_2048 || \
+			   PSA_NEED_OBERON_RSA_KEY_SIZE_3072 || \
+			   PSA_NEED_OBERON_RSA_KEY_SIZE_4096 || \
+			   PSA_NEED_OBERON_RSA_KEY_SIZE_6144 || \
+			   PSA_NEED_OBERON_RSA_KEY_SIZE_8192
+
```

```suggestion
	depends on PSA_NEED_OBERON_RSA_KEY_SIZE_ANY
	
config PSA_NEED_OBERON_RSA_KEY_SIZE_ANY
 bool default y
 depends on \
 PSA_NEED_OBERON_RSA_KEY_SIZE_1024 || \
 PSA_NEED_OBERON_RSA_KEY_SIZE_1536 || \
 PSA_NEED_OBERON_RSA_KEY_SIZE_2048 || \
 PSA_NEED_OBERON_RSA_KEY_SIZE_3072 || \
 PSA_NEED_OBERON_RSA_KEY_SIZE_4096 || \
 PSA_NEED_OBERON_RSA_KEY_SIZE_6144 || \
 PSA_NEED_OBERON_RSA_KEY_SIZE_8192	
```

This ANY symbol could be used elsewhere as well.

---

I will add this, yes